LFS171x/fabric-material/basic-network/README.md

```diff
@@ -0,0 +1,14 @@
+## Basic Network Config
```

Comment:
We should also get the basic-network for free with the fabric-sample repository `git clone https://github.com/hyperledger/fabric-samples.git`